### PR TITLE
Training#5

### PR DESCRIPTION
**Pull request by szympie**

### CODE REVIEW

File: scripts/train_unet.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        train_evaluator.run(train_loader)
+        validation_evaluator.run(valid_loader)
+
+
+    @trainer.on(Events.EPOCH_COMPLETED)
+    def print_train_results(engine):
+        metrics = train_evaluator.state.metrics
+        print(f"Training Results - Epoch: {trainer.state.epoch}  "
+              f"Dice loss: {metrics['dice_loss']:.4f} "
+              f"Dice: {1-metrics['dice_loss']:.4f}")
```
Comment (myychal) [inline scripts/train_unet.py]: Why not just `metrics['dice_score']`?

File: scripts/train_unet.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+
+    @trainer.on(Events.EPOCH_COMPLETED)
+    def print_validation_results(engine):
+        metrics = validation_evaluator.state.metrics
+        print(f"Validation Results - Epoch: {trainer.state.epoch}  "
+              f"Dice loss: {metrics['dice_loss']:.4f} "
+              f"Dice: {1-metrics['dice_loss']:.4f}")
```
Comment (myychal) [inline scripts/train_unet.py]: Same as for line 201

File: scripts/train_unet.py (hunk 1 above)
Comment (szympie) [reply scripts/train_unet.py]: +1

File: scripts/train_unet.py (hunk 2 above)
Comment (szympie) [reply scripts/train_unet.py]: +1

**Pull request merged**